crypto/merklekeystore/keystore.go

```diff
@@ -152,8 +152,10 @@ func New(firstValid, lastValid, interval uint64, sigAlgoType crypto.AlgorithmTyp
 
 // GetVerifier can be used to store the commitment and verifier for this signer.
 func (s *Signer) GetVerifier() *Verifier {
+	root := [112]byte{}
```

Comment:
could we use a constant value for that?

Reply:
I'm not sure I follow, const cannot be an array.